plugins/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -187,6 +187,13 @@
       - Service image path and tag.
       - Corresponds to the C(IMAGE) parameter of C(docker service create).
     type: str
+  init:
+    description:
+      - Use an init inside each service container to forward signals and reap processes.
+      - Corresponds to the C(--init) option of C(docker service create).
+      - Requires API version >= 1.37.
+    type: bool
+    version_added: "2.10"
```

Comment:
```suggestion
```
Please remove this line. We're adding `version_added` later once the next value is determined.